frontend/components/project/history/values/ValueText.vue

```diff
@@ -30,6 +24,11 @@ export default {
       required: true,
     },
   },
+  methods: {
+    isNothing(value) {
+      return value === null || value === ''
```

Comment:
perhaps should contain empty lists [] ?

Reply:
ValueText component only receive simple texts or null, ValueTags is handling lists..